packages/vats/src/bridge.js

```diff
@@ -3,6 +3,27 @@ import { E } from '@endo/far';
 
 const { Fail, details: X } = assert;
 
+/**
+ * Helper to type the registered scoped bridge correctly.
+ *
+ * FIXME: This is needed because `register` is not an async function, so
```

Comment:
interesting. it does seem like we would get it working without this but I agree it's not worth the time right now and this is an appropriate working solution